samples/drivers/video/tcpserversink/Kconfig

```diff
@@ -68,6 +68,11 @@ config VIDEO_CTRL_VFLIP
 	help
 	  If set, mirror the video frame vertically
 
+config VIDEO_ENCODED_PIXEL_FORMAT
+	string "Pixel format of the encoded frame"
+	help
+	  Pixel format of the encoded frame.
```

Comment:
this help text is of no use what so ever and just mirrors the prompt text, help text should be useful and guide the use to know what to put here, right now I don't know if I should put "composite", "tv", "rgb", "yuv420p" or "my dog has a bone what should I do?" or something else entirely